flake.nix

```diff
@@ -2,64 +2,29 @@
   description = "liqwid-plutarch-extra";
 
   inputs = {
-    nixpkgs.follows = "plutarch/nixpkgs";
+    nixpkgs.follows = "liqwid-nix/nixpkgs";
     nixpkgs-latest.url = "github:NixOS/nixpkgs";
-    # temporary fix for nix versions that have the transitive follows bug
-    # see https://github.com/NixOS/nix/issues/6013
-    nixpkgs-2111 = { url = "github:NixOS/nixpkgs/nixpkgs-21.11-darwin"; };
-    nixpkgs-2205 = { url = "github:NixOS/nixpkgs/22.05"; };
 
-    haskell-nix-extra-hackage.follows = "plutarch/haskell-nix-extra-hackage";
-    haskell-nix.follows = "plutarch/haskell-nix";
-    iohk-nix.follows = "plutarch/iohk-nix";
-    haskell-language-server.follows = "plutarch/haskell-language-server";
-
-    # Plutarch and its friends
-    plutarch = {
-      url = "github:Plutonomicon/plutarch-plutus";
-      inputs.emanote.follows =
-        "plutarch/haskell-nix/nixpkgs-unstable";
-      inputs.nixpkgs.follows =
-        "plutarch/haskell-nix/nixpkgs-unstable";
+    liqwid-nix = {
+      url = "/home/emi/work/liqwid/liqwid-nix";
```

Comment:
:eyes:

Reply:
Yep, haha. It's much more efficient for me to test repos  locally like this. Though, I'm thinking maybe it's possible to do some clever testing of repos "all at once" somehow. Especially since versioning is handled centrally in liqwid-nix now.